pkg/util/testing/wrappers.go

```diff
@@ -395,6 +395,19 @@ func MakeFlavor(rf, min string) *FlavorWrapper {
 	}}
 }
 
+// MakeFlavorMinMax creates a wrapper for a resource flavor with both
+// min and max specified.
+func MakeFlavorMinMax(rf, min string, max string) *FlavorWrapper {
```

Comment:
I heavily refactored wrappers in #604, so this change might not be necessary in the future.

Reply:
Removed.